ramls/event.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Event data model",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "id": {
+      "description": "UUID",
+      "type": "string"
+    },
+    "eventType": {
+      "description": "Unique Event type, human-readable String, not UUID",
+      "type": "string"
+    },
+    "metadata": {
+      "description": "Event metadata",
+      "type": "object",
+      "properties": {
+        "tenantId" : {
```

Comment:
Why is a tenant ID needed in the properties? Would this not be conveyed by the use of the standard header?

Reply:
This event will be put into topic as is, so it should contain all the necessary information. It might be tedious extracting tenantId from the header every time constructing a message.

Reply:
Translating an event to a topic sounds to me like an implementation detail.

To me, this design puts the onus on the client (to add a header and a property, in a non standard way) in order to simplify an internal part of an implementation (reading from a property rather than a header)

There may well be details about this area I don’t understand, so feel free to disregard this feedback.

Reply:
@TarasSpashchenko Ok. I was only raising awareness of the tradeoff I perceived between API simplicity and implementation convenience